egs/chime4/s5_1ch/local/chain/run_tdnn_lstm_recog.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+set -e -o pipefail
+
+stage=0
+nj=30
+train=noisy
+enhan=$1
+mdir=$2
+train_set=tr05_multi_${train}
+test_sets="dt05_real_$enhan dt05_simu_$enhan et05_real_$enhan et05_simu_$enhan"
+gmm=tri3b_tr05_multi_${train} # this is the source gmm-dir that we'll use for alignments; it
+                              # should have alignments for the specified training data.
+nnet3_affix=       # affix for exp dirs, e.g. it was _cleaned in tedlium.
+
+# Options which are not passed through to run_ivector_common.sh
+affix=1a  #affix for TDNN+LSTM directory e.g. "1a" or "1b", in case we change the configuration.
+common_egs_dir=
+reporting_email=
+
+# training chunk-options
+chunk_width=140,100,160
+# we don't need extra left/right context for TDNN systems.
+chunk_left_context=0
+chunk_right_context=0
+
+#decode options
+test_online_decoding=false  # if true, it will run the last decoding stage.
+
+# End configuration section.
+echo "$0 $@"  # Print the command line for logging
+
+
+. ./cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+
+if ! cuda-compiled; then
+  cat <<EOF && exit 1
+This script is intended to be used with GPUs but you have not compiled Kaldi with CUDA
+If you want to use GPUs (and have them), go to src/, and configure and make on a machine
+where "nvcc" is installed.
+EOF
+fi
+
+if [ $# -ne 2 ]; then
+  printf "\nUSAGE: %s <enhancement method> <model dir>\n\n" `basename $0`
+  echo "First argument specifies a unique name for different enhancement method"
+  echo "Second argument specifies acoustic and language model directory"
```

Comment:
(for next time you update: always give an example command line in usage messages-- it tends to be the most useful part of the usage message).